service/interfaces.go

```diff
@@ -31,3 +33,9 @@ type HealthChecker interface {
 	Stop()
 	AddCheck(name string, checker healthcheck.Checker) (err error)
 }
+
+//PermissionsStore defines the required methods from Mongo
```

Comment:
Again a really nit picky comment:  This Interface defines the behaviour of a  _PermissionsStore_. Users of this interface don't & shouldn't need to know the actual implementation is backed by MongoDB, by revealing too much about the inner workings you risk tightly coupling things together.